CHANGES.md

```diff
@@ -56,8 +56,8 @@
 
 ## I/Os
 
-* Support for X source added (Java/Python) ([BEAM-X](https://issues.apache.org/jira/browse/BEAM-X)).
-* Support for reading from Snowflake added (Java) ([BEAM-9722](https://issues.apache.org/jira/browse/BEAM-9722)).
```

Comment:
Should we go ahead and add a section for 2.23.0 and move this there so it doesn't get lost?